onshape-java/api-base/src/main/java/com/onshape/api/base/BaseClient.java

```diff
@@ -331,7 +331,7 @@ public void setWorkingDir(File workingDir) {
             }
             throw new OnshapeException("No entity in response");
         }
-        if (response.getMediaType().toString().startsWith(MediaType.APPLICATION_JSON)) {
+        if (response.getMediaType().toString().startsWith(MediaType.APPLICATION_JSON) | response.getMediaType().toString().startsWith("application/vnd.onshape.v2+json")) {
```

Comment:
Please create a constant/final static for the "application/vnd.onshape.v2+json" string and use the constant through the file.

Reply:
Good point, done. Should "v1" still be an option or is all json now "v2"?

Reply:
I wouldn't worry about v1 - the current stuff is using v2 and that's all the client needs to work with.